bigbluebutton-tests/playwright/learningdashboard/learningdashboard.js

```diff
@@ -0,0 +1,123 @@
+const { MultiUsers } = require("../user/multiusers");
+const e = require('../core/elements');
+const util = require('../polling/util.js');
+const { openChat } = require('../chat/util');
+const { expect } = require("@playwright/test");
+const Page = require("../core/page");
+const { sleep } = require("../core/helpers");
+const { ELEMENT_WAIT_LONGER_TIME } = require("../core/constants");
+const { openPoll } = require("./util");
+
+class LearningDashboard extends MultiUsers {
+  constructor(browser, context) {
+    super(browser, context);
+  }
+
+  async getDashboardPage(context) {
+    await this.modPage.waitAndClick(e.manageUsers);
+
+    const [dashboardPage] = await Promise.all([
+      context.waitForEvent('page'),
+      this.modPage.waitAndClick(e.learningDashboard),
+    ]);
+
+    await expect(dashboardPage).toHaveTitle(/Dashboard/);
+
+    this.dashboardPage = new Page(context, dashboardPage);
+  }
+
+  async writeOnPublicChat() {
+    await openChat(this.modPage);
+    await this.modPage.checkElementCount(e.chatUserMessageText, 0);
+
+    await this.modPage.type(e.chatBox, e.message);
+    await this.modPage.waitAndClick(e.sendButton);
+    await this.modPage.checkElementCount(e.chatUserMessageText, 1);
+
+    await this.dashboardPage.hasText(e.messageLearningDashboard, '1', ELEMENT_WAIT_LONGER_TIME);
+    
+  }
+
+  async userTimeOnMeeting() {
+    await this.modPage.waitAndClick(e.recordingIndicator);
+    await this.modPage.waitAndClick(e.confirmRecording);
+    await this.modPage.hasText(e.recordingIndicator, '00:0000:00');
+
+    const timeTest = await (this.dashboardPage.getLocator(e.userOnlineTime)).textContent();
+
+    const [hours, minutes, seconds] = timeTest.split(':').map(Number);
+    const timeInSeconds =  hours * 3600 + minutes * 60 + seconds;
```

Comment:
I think we can add a unique function to get the time in second, avoiding duplications